src/main/scala/com/atomist/tree/marshal/LinkedJsonTreeDeserializer.scala

```diff
@@ -103,6 +102,21 @@ object LinkedJsonTreeDeserializer extends LazyLogging {
     }
 }
 
+class EmptyLinkableContainerTreeNode() extends ContainerTreeNode {
+
+  override def nodeName: String = "empty"
```

Comment:
I think it's probably helpful to have this here as it makes it obvious if you encounter a node of this type whilst debugging. There is also a precedent of doing this elsewhere in rug that I followed.